doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -530,14 +530,16 @@ The current |NCS| main branch is based on the Zephyr v2.7 development branch.
 Matter (Project CHIP)
 =====================
 
-The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``9012f08de9b7340e7d59d51a7ec8a6cdcfda9d15``.
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``bbd19d92f6d58ef79c98793fe0dfb2979db6336d``.
 
 The following list summarizes the most important changes inherited from the upstream Matter:
 
 * Added:
 
   * Support for Administrator Commissioning Cluster, which allows enabling or disabling the commissioning window on a Matter device.
     This is required by the Matter multi-admin functionality.
+  * Support for Power Source Cluster, which exposes information about the power source of a Matter device, including the battery level.
+  * Initial support for Thread Sleepy End Devices.
```

Comment:
Would it make sense at this point to move the contents of this section under Protocols/Matter? We could then have just an entry about these changes "Updated the Matter fork in the |NCS|... [...]. The following lists summarizes [...]"?

Reply:
To be honest, I don't understand the structure of the changelog. Isn't the "Protocols" section meant to be used for changes in sdk-nrf repository related to the protocols? Otherwise, we would we have two places for changes in the same project, such as Matter?

Reply:
Yes to question 1). As for question 2), my proposal comes from the fact that for Zephyr we avoid listing specific changes from upmerge. I think we could also skip listing them for Matter, if we provide the hash of the implemented changes.

Reply:
I see. I think we may not list the changes for Zephyr as there are too many of them :). For MCUBoot we have one change listed, so it seems occasionally used. We can talk about as I'm not against skipping it for Matter, but on the other hand, maybe it is useful for someone as the upstream Matter doesn't publish release notes yet.